### DISCUSSION TITLE
What is the ‘use client’ used for？

### DISCUSSION THREAD

**Discussion by wh5938316** [vercel/next.js#41828] [Help] [answered]
next.js 13 with `turbopack` example project: https://github.com/vercel/next.js/blob/canary/examples/with-turbopack/app/context/CategoryNav.tsx

What is the ‘use client’ in jsx file head used for?

<img width="547" alt="image" src="https://user-images.githubusercontent.com/16160933/197956665-389febb8-86a8-4af5-b884-85f585c59a8d.png">

**Comment by ghost** [comment, marked as answer]
Hello @wh5938316, for the time being the docs are still in beta, you can read here all about Client Components : https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components

**Reply by wh5938316** [reply]
thanks

**Comment by icyJoseph** [comment]
By default when using `app`, and independently of turbopack's usage, things are rendered server side first, unless you introduce the client directive `use client` at the top of the file. 

When you do, JS-driven interactivity is allowed, think of state, effects, etc.
